packages/react-composites/src/composites/CallWithChatComposite/components/MoreDrawer.tsx

```diff
@@ -68,6 +80,8 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
     [speakers, onSelectSpeaker]
   );
 
+  const options = inferCallWithChatControlOptions(props.callControls);
```

Comment:
`options` is a little vague here, why not something like `drawerSelectionOptions`? since we will want to add to this in the future (start recording and such)